Makefile

```diff
@@ -23,7 +23,7 @@ clean:
 	-rm -f bin/release
 
 package: clean build
-	@tar cvzf procfile-cnb-$(VERSION).tgz bin/ lib/ README.md LICENSE buildpack.toml
+	@pack buildpack package --format=file heroku_procfile_$(VERSION).cnb
```

Comment:
Travis CI is failing on master after this change (it hadn't run when this originally landed, I manually triggered CI now):
`/bin/bash: pack: command not found`
https://travis-ci.com/github/heroku/procfile-cnb/builds/234934834#L327

Reply:
I've created an epic and some work items for fixing this, migrating to Circle CI and the languages team officially adopting this repo:
https://gus.lightning.force.com/lightning/r/ADM_Epic__c/a3QAH0000000OaQ2AU/related/Work__r/view